### DISCUSSION TITLE
send photo/document with file_id

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#146] [General]
HI
I want send photo or document with file id but just can send in storage
how can send with file id?

**Comment by ah-rahimi** [comment]
@fabio-ivona

**Comment by fabio-ivona** [comment]
Not supported yet, it's on our Todo list

**Reply by ah-rahimi** [reply]
Please put in high priority
I need it very much

**Reply by ah-rahimi** [reply]
You can do this by defining another method @fabio-ivona

**Reply by fabio-ivona** [reply]
This is an open source project, so all features are implemented when contributors have time and will to do it. If you have stricter timings, feel free to submit a PR for it and I'll review it as soon as possible

**Reply by fabio-ivona** [reply]
implemented
